Source/XUnitExtensions/Lit/ILitCommand.cs

```diff
@@ -2,12 +2,31 @@
 using System.Collections.Generic;
```

Comment:
What does this change to Lit have to do with the PR?

Reply:
The PR uses path expansion "%S/**/*.java" in the Lit commands that wasn't well supported by our Lit emulation.